src/Common/tests/System/Collections/IList.Generic.Tests.cs

```diff
@@ -608,5 +608,50 @@ public void IList_Generic_RemoveAt_ZeroMultipleTimes(int count)
         }
 
         #endregion
+
+        #region Enumerator.Current
+
+        // Enumerator.Current should fail at end after new elements was added
```

Comment:
This comment isn't always correct. The failure of Enumerator.Current is dependent on `Enumerator_Current_UndefinedOperation_Throws` in the below test.